reference/constraints/Uuid.rst

```diff
@@ -117,6 +117,7 @@ The following PHP constants can also be used:
 
 All five versions are allowed by default.
 
+.. include:: /reference/constraints/_ltrim-rtim-options.rst.inc
```

Comment:
Typo in the file name -> should be `rtrim`, not `rtim` (to be fixed eveywhere :))